ecleankernel/__main__.py

```diff
@@ -256,6 +261,34 @@ def main(argv: typing.List[str]) -> int:
                 bootloader=bootloader,
                 destructive=args.destructive)
 
+            if HOOK_DIR.is_dir():
+                preserved_kernels = [k for k in kernels
+                                     if k not in removals]
+
+                def as_version_list(kernels: List[Kernel]) -> str:
+                    return " ".join(list(map(lambda x: x.version, kernels)))
+
+                env = dict()
```

Comment:
You should probably copy `os.environ`, otherwise you're going to run hooks without `PATH` set.